export function consoleLog() {
  console.log('WITHIN SOURCE')
}

export function throwError() {
  throw new Error('WITHIN SOURCE')
}
